paddle/fluid/operators/fused/mkldnn/fusion_gru_mkldnn_op.cc

```diff
@@ -246,7 +246,7 @@ class GRUMKLDNNHandler : public platform::MKLDNNHandlerT<T, dnnl::gru_forward> {
       memory_p = std::make_shared<dnnl::memory>(this->fwd_pd_->src_iter_desc(),
                                                 this->engine_);
 
-      dnnl::stream astream(this->engine_);
+      auto astream = paddle::platform::MKLDNNDeviceContext::tls().get_stream();
```

Comment:
Please consider this change to make sure no copy is done.
```suggestion
      const auto& astream = paddle::platform::MKLDNNDeviceContext::tls().get_stream();
```
The `execute` function accepts `astream` arg as a [const ref already](https://spec.oneapi.com/versions/latest/elements/oneDNN/source/primitives/reorder.html#_CPPv4NK4dnnl7reorder7executeERK6streamR6memoryR6memory).

Reply:
ok . done